server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/DownloadGet.feature

```diff
@@ -69,6 +69,12 @@ Feature: Download GET
     Then she can read that blob
     And the blob size is 36
 
+  Scenario: Getting a message then getting its Content-Type of the blob
+    Given "alice@domain.tld" has a message "m1" in "INBOX" mailbox with subject "my test subject", content "testmail"
+    And "alice@domain.tld" ask for messages "m1"
+    When "alice@domain.tld" downloads should not return Content-Type of the message by its blobId
```

Comment:
This sentence doesn't make sense: it mixes "When" (the action phase") with an assertion "should not return Content-type".
Please split it in two.
Btw, we don't care to keep the last line.